packages/grid/x-data-grid-generator/src/renderer/renderTotalPrice.tsx

```diff
@@ -13,10 +13,14 @@ const Value = styled('div')(({ theme }) => ({
   alignItems: 'center',
   justifyContent: 'flex-end',
   '&.good': {
-    backgroundColor: alpha(theme.palette.success.main, 0.3),
+    backgroundColor: theme.vars
+      ? `rgba(${theme.vars.palette.success.main} /  0.3)`
```

Comment:
```suggestion
      ? `rgba(${theme.vars.palette.success.mainChannel} /  0.3)`
```

@alexfauquette the `mainChannel` is the new token that Material UI generates if the consumers use the new provider.

It is required because this is the only way to use color with transparency:
```js
`rgba(${theme.vars.palette.success.main} / 0.3)`
// rgba(#the-hex-color / 0.3) is invalid in CSS ❌

`rgba(${theme.vars.palette.success.mainChannel} / 0.3)`
// rgba(83 38 153 / 0.3) ✅
```